src/Tools/FirstRunCertGenerator/test/CertificateManagerTests.cs

```diff
@@ -25,7 +25,12 @@ public CertificateManagerTests(ITestOutputHelper output, CertFixture fixture)
         Output = output;
     }
 
-    public const string TestCertificateSubject = "CN=aspnet.test";
+    private const string TestCertificateSubject = "CN=aspnet.test";
+
+    // See comment near the definition of AspNetHttpsOid in CertificateManager.
```

Comment:
Is it too hard/ugly to pull in/reference the CertificateGeneration const's here directly to have a single source of the values?  Its already in shared sources, so maybe have a CertConsts public class?

Reply:
Done. Just made the dev-certs internals visible to the test project.